packages/knip/src/plugins/lefthook/index.ts

```diff
@@ -11,11 +12,22 @@ const title = 'Lefthook';
 
 const enablers = ['lefthook', '@arkweid/lefthook', '@evilmartians/lefthook'];
 
-const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
+const patterns = [
+  'lefthook.{yml,yaml,json,toml}',
+  '.lefthook.{yml,yaml,json,toml}',
+  'lefthook-local.{yml,yaml,json,toml}',
+  '.lefthook-local.{yml,yaml,json,toml}',
+  '.config/lefthook.{yml,yaml,json,toml}',
+];
+
+const isEnabled: IsPluginEnabled = async ({ dependencies, cwd }) =>
+  Boolean(hasDependency(dependencies, enablers) || (await _firstGlob({ cwd, patterns })));
```

Comment:
The issue with enabling the plugin by config file existence is that we'll no longer be able to determine the opposite: how can we mark the dependency as unused in the future (as it is still often listed)?